index.js

```diff
@@ -31,18 +31,16 @@ var noaaForecaster = {
    * @param forecastDetails {Object} - NOAA api params describing what sort of environmental data to retrieve
    */
   attachForecasts: function (geoBusinessObjects, forecastDetails) {
+    var chunkSize = 200;
     if (!this._token) { return BluebirdPromise.reject('Must call setToken before calling get.'); }
     forecastDetails = this._addDefaultOptions(forecastDetails);
 
     return this._validateGeoObjects(geoBusinessObjects).bind(this)
       .then(function() {
-        var listLatLon = this._getLatLonList(geoBusinessObjects);
-        var options = _.extend({}, forecastDetails, { listLatLon: listLatLon});
-        var url = this._convertOptionsToUrlString('data', options);
-        return this._makeCall(url);
+        return this._getForecastPerPoint(geoBusinessObjects, forecastDetails, chunkSize);
```

Comment:
I think the chunkSize could be an internal detail to the `getForecastPerPoint` function.  That function should fetch in chunks and return a flattened version of the results.  The next function in the chain doesn't have to know or care that there's a chunk limitation in the `_getForecastPerPoint` process.